src/server/controllers/index.js

```diff
@@ -23,7 +23,6 @@ Router.get('/overview', getCount.get);
 // Participants Routes
 Router.get('/participants', participants.get);
 Router.get('/participant/:id', participants.getDetails);
-Router.post('/participants/search/forename', participants.searchByForeName);
 Router.post('/participants/search/surname', participants.searchBySurName);
```

Comment:
@AsalaKM Nice job with deleting all of the forename code now that it's not used anymore.  Removing unused code ensures that you won't waste time refactoring it or maintaining it later, when the fact that it's not used by any client gets forgotten :)

I would suggest updating the endpoint url to reflect the fact that it's a name, not a surname, search now.  I think that will help avoid some confusion down the road.

Reply:
Thanks @tomduggan85 :smile: I do it in the  #279 :+1: